input-github-username.sh

```diff
@@ -0,0 +1,16 @@
+set -u;
+if ! gp env | grep -q 'GITHUB_USERNAME'; then {
+  until read -r -p "++++ Enter your Github username (only required once): " GITHUB_USERNAME; do continue; done
+  gp env GITHUB_USERNAME="$GITHUB_USERNAME"; # Sets this variable on https://gitpod.io/variables which can be used later on
+} fi
+
+if test ! -v GITHUB_USERNAME; then {
+  eval $(gp env -e);
+} fi
+
+
+git -C /workspace/mattermost-server remote add upstream https://github.com/mattermost/mattermost-server.git
+git -C /workspace/mattermost-server remote set-url origin https://github.com/${GITHUB_USERNAME}/mattermost-server.git
+
+git -C /workspace/mattermost-webapp remote add upstream https://github.com/mattermost/mattermost-webapp.git
+git -C /workspace/mattermost-webapp remote set-url origin https://github.com/${GITHUB_USERNAME}/mattermost-webapp.git
```

Comment:
I put them in the same bash file because when splitted second script doesnt (i.e. setting remote url script) see `GITHUB_USERNAME` as empty string. I am sure there is a solution for that but I couldnt find. Please let me know if you have any suggestion

Reply:
You can do
```bash
source input-github-username.sh
./your-other-script.sh
```

Reply:
Thanks. I splitted into two by calling other script inside first script.